### PR TITLE
feat : like/dislike functionality added

### PR DESCRIPTION
**Pull request by vikrantkode**
### Like / Dislike functionality added
        after like it goes to liked page and clicking upon there will remove from like page

### CODE REVIEW

**Comment by ashifkhn** [comment]
ui looks clean, features working.

File: src/components/LikeCard/ViLikeCard.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
+
+:root {
+    --hover-color: #f3f1f1f3;
+    --shadow-color: black;
+    --card-subheadcolor: gray;
```
Comment (sharmayatendra) [inline src/components/LikeCard/ViLikeCard.css:5]: use color variables

File: src/components/LikeCard/ViLikeCard.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    font-size: 2rem;
+}
+
+.vicons {
+    margin-right: 0.5rem;
+    cursor: pointer;
+}
+
+.vicons:hover {
+    background-color: #f5dadaf3;
```
Comment (sharmayatendra) [inline src/components/LikeCard/ViLikeCard.css]: same as above

File: src/components/LikeCard/ViLikeCard.css (hunk 1 above)
Comment (sharmayatendra) [inline src/components/LikeCard/ViLikeCard.css:5]: either directly give name or use hex

File: src/components/LikeCard/ViLikeCard.css (hunk 2 above)
Comment (vikrantkode) [reply src/components/LikeCard/ViLikeCard.css]: changes done

File: src/components/VideoCard/ViCard.css [hunk 3]
```diff
@@ -74,5 +74,8 @@
 .vicons {
     margin-right: 0.5rem;
     cursor: pointer;
+}
 
+.vicons:hover {
+    background-color: #f5dadaf3;
```
Comment (sharmayatendra) [inline src/components/VideoCard/ViCard.css:80]: dont hard code color

**Review by sharmayatendra — APPROVED** [review]
good work
good naming convention
good to merge

**Pull request merged**